docs/designers-developers/developers/filters/i18n-filters.md

```diff
@@ -0,0 +1,109 @@
+# i18n Filters
+
+The i18n functions (`__()`, `_x()`, `_n()` and `_nx()`) provide translations of strings for use in your code. The values returned by these functions are filterable if you need to override them, using the following filters:
+
+- `i18n.gettext`
+- `i18n.gettext_with_context`
+- `i18n.ngettext`
+- `i18n.ngettext_with_context`
+
+## Filter Arguments
+
+The filters are passed the following arguments, in line with their PHP equivalents.
+
+### i18n.gettext
+
+```jsx
+function i18nGettextCallback( translation, text, domain ) {
+   return translation; 
+}
+```
+
+### i18n.gettext_with_context
+
+```jsx
+function i18nGettextWithContextCallback( translation, text, context, domain ) {
+   return translation; 
+}
+```
+
+### i18n.ngettext
+
+```jsx
+function i18nNgettextCallback( translation, single, plural, number, domain ) {
+	return translation;
+}
+````
+
+### i18n.ngettext_with_context
+
+```jsx
+function i18nNgettextWithContextCallback(
+	translation,
+	single,
+	plural,
+	number,
+	context,
+	domain
+) {
+	return translation;
+}
+````
+
+## Basic Example
+
+Here is a simple example, using the `i18n.gettext` filter to override a specific translation.
+
+```jsx
+// Define our filter callback.
+function myPluginGettextFilter( translation, text, domain ) {
+	if ( text === 'Add to Reusable blocks' ) {
+		return 'Save to MyOrg block library';
+	}
+	return translation;
+}
+
+// Adding the filter
+wp.hooks.addFilter(
+		'i18n.gettext',
+		'my-plugin',
```

Comment:
The filter identifier should be unique so something less common would be preferable:
```suggestion
		'my-plugin/override-add-to-reusable-blocks-label',
```

Reply:
Updated. Thanks.